rosa_tutorials/rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial.adoc

```diff
@@ -0,0 +1,175 @@
+:_content-type: ASSEMBLY
+[id=“rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial”]
+= Tutorial: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-<topic>-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2022-04-26'
+//title: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry (ECR)
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Kevin Collins
+//  - Byron Miller
+//---
+== Prerequisites
+
+* link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[AWS CLI]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/ocp/[`oc` CLI] 4.11+
+* link:https://podman-desktop.io/[Podman Desktop]
+
+== Background
+
+Quick introduction by Ryan Niksch and Charlotte Fung on link:https://youtu.be/1PBFtpCIMBo[YouTube].
+
+There are two options to use to authenticate with AWS Elastic Container Registry (ECR) to pull images.
+
+The traditional method is to create a pull secret for ECR:
+
+.Example
+[source,terminal]
+----
+$ oc create secret docker-registry ecr-pull-secret  \
+  --docker-server=<registry id>.dkr.ecr.<region>.amazonaws.com  \
+  --docker-username=AWS --docker-password=$(aws ecr get-login-password) \
+  --namespace=hello-world
+----
+
+However Amazon ECR tokens expire every 12 hours which means you need to re-authenticate every 12 hours either through scripting or do so manually.
+
+A second, and preferred method, is to attach an ECR policy to your cluster's worker machine profiles which this guide will walk you through.
+
+== Attach ECR policy role
+
+You can attach an ECR policy to your cluster, giving the cluster permissions to pull images from your registries. ROSA worker machine instances include pre-defined IAM roles, named differently depending on whether the cluster is an STS cluster or a non-STS cluster.
+
+=== STS cluster role
+
+`ManagedOpenShift-Worker-Role` is the IAM role attached to Red Hat OpenShift Service on AWS (ROSA) STS compute instances.
```

Comment:
`ManagedOpenShift-Worker-Role`  is the default rosa attached to the cluster. But once customer customized the value, it won't work anymore. Better to provide the command to grep their role. Below command can work. But if anybody has a better way, feel free to provide
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa describe cluster -c $CLUSTER|grep Worker-Role|tr " " "\n"|grep Worker-Role
arn:aws:iam::301721915996:role/sdq-ci-lgxsu-Worker-Role
```